Dockerfile

```diff
@@ -74,3 +79,47 @@ RUN	ln -s /opt/local/lib/livepeer-w3/livepeer-w3.js /usr/local/bin/livepeer-w3 &
 EXPOSE	1935	4242	8080	8889/udp
 
 CMD	["/usr/local/bin/MistController", "-c", "/etc/livepeer/catalyst.json"]
+
+FROM catalyst AS livepeer-in-a-box
+
+ARG TARGETARCH
+
+RUN	apt update && apt install -yqq \
+	rabbitmq-server \
+	nginx \
+	gdb \
+	inotify-tools \
+	file \
+	# for `shasum`
+	perl \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN curl -L -O https://binaries.cockroachdb.com/cockroach-v23.1.5.linux-$TARGETARCH.tgz \
+	&& tar xzvf cockroach-v23.1.5.linux-$TARGETARCH.tgz \
+	&& mv cockroach-v23.1.5.linux-$TARGETARCH/cockroach /usr/bin/cockroach \
+	&& rm -rf cockroach-v23.1.5.linux-$TARGETARCH.tgz cockroach-v23.1.5.linux-$TARGETARCH \
+	&& cockroach --version
+
+RUN curl -o /usr/bin/minio https://dl.min.io/server/minio/release/linux-$TARGETARCH/minio \
+	&& curl -o /usr/bin/mc https://dl.min.io/client/mc/release/linux-$TARGETARCH/mc \
+	&& chmod +x /usr/bin/minio /usr/bin/mc \
+	&& minio --version \
+	&& mc --version
+
+ADD ./scripts /usr/local/bin
+ADD ./config/full-stack.json /etc/livepeer/full-stack.json
+
+ENV CATALYST_DOWNLOADER_PATH=/usr/local/bin \
+	CATALYST_DOWNLOADER_MANIFEST=/etc/livepeer/manifest.yaml \
+	CATALYST_DOWNLOADER_UPDATE_MANIFEST=true \
+	COCKROACH_DB_SNAPSHOT=https://github.com/iameli-streams/livepeer-in-a-box-database-snapshots/raw/f59e7ce7a631dbcd176580a54b4fe5f31f9e4dbc/livepeer-studio-bootstrap.tar.gz
```

Comment:
Why livepeer-in-a-box-database-snapshots isn't stored in catalyst repo directly where livepeer-in-a-box is defined?

Also the dump contains `logs/` directory which could be dropped to reduce the size of dump if that matters.

Reply:
Just 'cause they might be big; I don't want to put them in the git history. We should be moving to database snapshots instead of filesystem dumps anyway which will make the issue not matter anymore.